openlibrary/templates/lib/covers.html

```diff
@@ -27,6 +27,9 @@
     carousel.add(index, "<h3>Loading books...</h3>");
     page.loadPage(index-1, function(data) {
         carousel.add(index, page.renderCovers(data));
+
+        console.log('availability about to get called...');
+        updateBookAvailability('#coversCarousel li);
```

Comment:
Syntax error? Missing closing single-quote? (GitHub shows this line and following code with warning red syntax coloring...)